frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-details.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { DetailsPage } from '@console/internal/components/factory';
+import {
+  navFactory,
+  SectionHeading,
+  ResourceSummary,
+  Kebab,
+} from '@console/internal/components/utils';
+import { CatalogSourceListPage } from '../catalog-source';
+
+const OperatorHubDetails = ({ obj }) => (
+  <div className="co-m-pane__body">
+    <SectionHeading text="OperatorHub Overview" />
+    <ResourceSummary resource={obj} podSelector="spec.podSelector" showNodeSelector={false} />
+  </div>
+);
+
+const SourcesPage = {
+  href: 'sources',
+  name: 'Sources',
+  component: (props) => <CatalogSourceListPage showTitle={false} {...props} />,
+};
```

Comment:
I have a feeling inlining this will cause problems like in #2979. I'd pull this out

```tsx
const Sources = (props) => <CatalogSourceListPage showTitle={false} {...props} />;
const SourcesPage = {
  href: 'sources',
  name: 'Sources',
  component: Sources,
};
```